Tarot Reader/tarot_card_reader.py

```diff
@@ -0,0 +1,177 @@
+import random
+
+
+# tarot deck of 78 cards and 'predictions'
+tarot_deck = [
```

Comment:
store this information in a text file and then load it here instead of hardcoding it in the code. Also, make sure that after loading the contents from the file, the code is not affected. Check if any text processing is required after loading the text

Reply:
I've created the text file `tarot.txt` and used simple file handling to load it. The code runs just as well, and I don't believe further text processing is required